src/styles/_card.scss

```diff
@@ -0,0 +1,135 @@
+@import 'variables';
+
+.card {
+  width: 200px;
+  background-color: $four-color;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  &__description {
+    width: 166px;
+    height: 36px;
+    font-weight: 500;
+    font-size: $font-size-md;
+    line-height: 18px;
+    color: $triple-color;
+
+    &--margin-top {
+      margin-top: 40px;
+    }
+  }
+
+  &__code {
+    width: 98px;
+    height: 14px;
+    font-weight: 400;
+    font-size: $font-size-sm;
+    line-height: 14px;
+    color: $secondary-color;
+
+    &--margin-top {
+      margin-top: 4px;
+    }
+  }
+
+  &__rating {
+    width: 166px;
+    height: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &--margin-top {
+      margin-top: 16px;
+    }
+  }
+
+  &__stars {
+    padding: 0;
+    display: flex;
+    gap: 4px;
+    list-style: none;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url('../images/star.svg');
+    background-size: contain;
+
+    &--active {
+      background-image: url('../images/star-active.svg');
```

Comment:
Verify that the path '../images/star-active.svg' is correct and the file exists. Incorrect paths will result in missing images.